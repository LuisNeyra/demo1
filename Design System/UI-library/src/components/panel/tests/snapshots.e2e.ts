import { snapIt } from '../../../test';

const component = 'panel';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('correctly with no props', '<blaze-panel>default</blaze-panel>');
    snap('correctly with height set', '<blaze-panel height="350">default</blaze-panel>');
  });
});
