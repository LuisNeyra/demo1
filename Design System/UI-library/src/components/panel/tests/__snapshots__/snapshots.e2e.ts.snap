// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`panel renders correctly with height set 1`] = `"<blaze-panel height=\\"350\\" class=\\"hydrated\\"><!----><div class=\\"o-panel-container\\" style=\\"height: 350px;\\"><div class=\\"o-panel\\">default</div></div></blaze-panel>"`;

exports[`panel renders correctly with no props 1`] = `"<blaze-panel class=\\"hydrated\\"><!----><div class=\\"o-panel-container\\" style=\\"height: 0px;\\"><div class=\\"o-panel\\">default</div></div></blaze-panel>"`;
